packages/dev/core/src/Culling/Helper/transformFeedbackBoundingHelper.ts

```diff
@@ -75,28 +78,9 @@ export class TransformFeedbackBoundingHelper implements IBoundingInfoHelperPlatf
                     attribs.push(VertexBuffer.MatricesIndicesExtraKind);
                     attribs.push(VertexBuffer.MatricesWeightsExtraKind);
                 }
-
-                const skeleton = mesh.skeleton;
-
                 defines.push("#define NUM_BONE_INFLUENCERS " + mesh.numBoneInfluencers);
-
-                if (skeleton.isUsingTextureForMatrices) {
-                    defines.push("#define BONETEXTURE");
-
-                    if (uniforms.indexOf("boneTextureWidth") === -1) {

```

Comment:
TransformFeedbackBoundingHelper had a lot of indexOf() === -1. This was probably a result of copying code from ShaderMaterial, so additional refactoring was done.